calpip/Task1/Task 1-1.txt

```diff
@@ -1,21 +1,21 @@
 Dear Sir or Madam,
 
-My name is Wellington and I work voluntarily in a community organization that 
-makes a wonderful job for local children. I am writing to ask your company for
```

Comment:
`makes a wonderful job` sounds awkward, you can say `does a wonderful job` to be correct

I changed it to `provides a wonderful service` to be more specific (job could be anything)